src/Juvix/Pipeline/Backend/Internal.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE LiberalTypeSynonyms #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeFamilyDependencies #-}
+{-# LANGUAGE UndecidableInstances #-}
```

Comment:
Are some of these not in our default pacakge?

Reply:
Some of them definitely should be if they aren't, though not UndecidableInstances